integration-tests/rita.py

```diff
@@ -78,12 +87,12 @@ def exec_or_exit(command, blocking=True, delay=0.01):
 
 
 def cleanup():
-    os.system("rm -rf *.log *.pid *.toml private-key*")
+    os.system("sudo rm -rf *.log *.pid *.toml private-key*")
```

Comment:
There's literally no point as rita.py is run with `sudo` in `rita.sh` at line 58